java/simulator/src/main/java/com/hazelcast/simulator/probes/impl/HdrProbe.java

```diff
@@ -25,8 +25,8 @@
  * HDR-Histogram implementation of the {@link Probe}.
  */
 public class HdrProbe implements Probe {
-    // we want to track up to an hour.
-    static final long HIGHEST_TRACKABLE_VALUE = HOURS.toNanos(1);
+    // we want to track up to 24-hour.
+    static final long HIGHEST_TRACKABLE_VALUE_NANOS = HOURS.toNanos(24);
```

Comment:
```suggestion
    static final long HIGHEST_TRACKABLE_VALUE_NANOS = TimeUnit.DAYS.toNanos(1);
```

nit - personal preference.